### DISCUSSION TITLE
Configuration Intégration

### DISCUSSION THREAD

**Discussion by chrsc59** [saniho/apiEnedis#219] [Ideas]
Ce serait bien d'avoir des libelles sur les champs à remplir lors de l'ajout de l'intégration.
Rien n'apparait

![image](https://user-images.githubusercontent.com/92226476/210783433-c8de55f9-4793-4003-81f6-2258d64862f3.png)

**Comment by superjaegermaster** [comment]
Pas eu de soucis pour moi en HAos...

Il faut donner plus d'infos :  quel OS, version de HA etc....
